app/mobile-content/exhibits/edit-exhibit-pages/edit-exhibit-pages.component.html

```diff
@@ -5,41 +5,61 @@
-      <md-option *ngFor="let statusOption of statusOptions" [value]="statusOption">{{ statusOption | translate }}</md-option>
-    </md-select>
+    <md-list md-line class="infopages" *ngIf="page.hasInfoPages()">
+      <h3 md-subheader>{{ 'additional information pages' | translate }}:</h3>
+      <md-list-item *ngFor="let infoPageId of page.additionalInformationPages; let idx = index; let isFirst = first; let isLast = last">
+        <h4 md-line *ngIf="infoPages.has(infoPageId)">
+          {{ infoPages.get(infoPageId).title || '(' + ('no title' | translate) + ')' }}
+        </h4>
+        <p md-line *ngIf="infoPages.has(infoPageId)">
+          <span>{{ infoPages.get(infoPageId).pageType | translate }}</span>
+          &nbsp;&bull;&nbsp;
+          <span *ngIf="infoPages.has(infoPageId)" [ngClass]="{'warning': !infoPages.get(infoPageId).isPublished()}">
+            {{ infoPages.get(infoPageId).status | translate }}
+          </span>
+        </p>
+        <p md-line *ngIf="infoPages.has(infoPageId)">{{ infoPages.get(infoPageId).text }}</p>
+        <button md-icon-button color="primary" *ngIf="infoPages.has(infoPageId)"
+                (click)="editPage(infoPages.get(infoPageId))" title="{{ 'edit' | translate }}">
+          <md-icon>edit</md-icon>
+        </button>
+      </md-list-item>
+    </md-list>
+    
+    <button md-icon-button color="primary" [disabled]="isLast" title="{{ 'move down' | translate }}" (click)="movePageDown(page)">
+      <md-icon>details</md-icon>
+    </button>
+    <button md-icon-button color="primary" [disabled]="isFirst" title="{{ 'move up' | translate }}" (click)="movePageUp(page)">
+      <md-icon>change_history</md-icon>
+    </button>
+    <button md-icon-button color="primary" (click)="editPage(page)" title="{{ 'edit page' | translate }}">
+      <md-icon>edit</md-icon>
+    </button>
+    <button md-icon-button color="warn" (click)="unsetPage(page)" title="{{ 'unset page' | translate }}">
+      <md-icon>close</md-icon>
+    </button>
+  </md-list-item>
+</md-list>
 
-    <div>
-      <button md-icon-button color="primary" [disabled]="isLast" title="{{ 'move down' | translate }}" (click)="movePageDown(page)">
-        <md-icon>details</md-icon>
-      </button>
-      <button md-icon-button color="primary" [disabled]="isFirst" title="{{ 'move up' | translate }}" (click)="movePageUp(page)">
-        <md-icon>change_history</md-icon>
-      </button>
-      <button md-icon-button color="primary" title="{{ 'save' | translate }}" [disabled]="page.text.trim().length === 0" (click)="savePage(page)">
-        <md-icon>save</md-icon>
-      </button>
-      <button md-icon-button color="warn" title="{{ 'delete' | translate }}" (click)="deletePage(page)">
-        <md-icon>delete_forever</md-icon>
-      </button>
-    </div>
-  </div>
-</div>
+<ng-template #noPages>
+  <div class="no-pages-message" *ngIf="pages != null && pages.length === 0">{{ 'no pages' | translate }}</div>
```

Comment:
This is an old JS trick. The `!=` operator, unlike `!==`, performs type coercion. So `pages != null` actually checks for both `null` and `undefined`, while `!==` only checks for whatever is on the right side. I just wanted to cover all bases since we are dealing with server responses here. In any case, even though our linter usually forbids `!=` and `==`,  it is configured to allow them for null checking.